### ISSUE TITLE
We need videos for installation instructions

### ISSUE THREAD

**Issue by alexAubin** [YunoHost/issues#1446]
Some work started during last's Brique Camp as well ... Ideally we need something like what Freedbom Box provides (c.f.  bottom of https://freedombox.org/)

**Comment by decentral1se** [comment]
Are videos already made? 

I know of https://peertube.boneheadmedia.com/accounts/root/videos at least ;)

**Comment by alexAubin** [comment]
There was some draft of videos yes, though not finished I believe ... Dunno where they ended up :/

**Comment by Plumf** [comment]
Does Yunohost have a graphic charter with color references, fonts used ? Yunohost has a peertube space, right ?

**Comment by alexAubin** [comment]
Yes we have a peertube space though my opinion is that we already have too many things to do to maintain a peertube server but that's another story

We don't really have a graphic charter ... I just guess that kind of aim to keep a relatively minimalist aesthetic though trying to keep it friendly and warm (and not too "profesionnaly cold" like some websites selling products are, I dunno). That's not very accurate though ¯\\\_(ツ)\_/¯

In case you're interested, I can also show you the draft of website from @e-lie (just not a big fond of sharing the link publicly because idk it's still a draft)

**Comment by Plumf** [comment]
While checking the PR of the doc I came across this app: https://magicmirror.builders/
As far as video can provide inspiration, so I paste link here: https://vimeo.com/171152845

**Comment by frju365** [comment]
For the peertube space, we have this  : https://cinema.yunohost.support (I maintain it and to be honnest, for instance, it's really easy). I gave an answer on the forum : https://forum.yunohost.org/t/appel-a-contributions-realisation-dun-tuto-installation-de-yunohost-realization-of-a-tutorial-installation-of-yunohost/9234/2?u=frju365
